flang/include/flang/Optimizer/Dialect/CUF/CUFOps.td

```diff
@@ -260,9 +263,27 @@ def cuf_KernelOp : cuf_Op<"kernel", [AttrSizedOperandSegments,
         ( `,` `stream` `=` $stream^ )? `>` `>` `>`
         custom<CUFKernelLoopControl>($region, $lowerbound, type($lowerbound),
             $upperbound, type($upperbound), $step, type($step))
+        `reduce_oprnds` $reduceOperands `:` type($reduceOperands)
```

Comment:
```suggestion
        `reduce` `(` $reduceOperands `:` type($reduceOperands) `)`
```

To look more like other operations in CUF/FIR/OpenACC dialects.  Also I would remove `_oprnds` since we know these are operands anyway.